core/transaction_pool/transaction_pool_error.cpp

```diff
@@ -8,8 +8,11 @@
 OUTCOME_CPP_DEFINE_CATEGORY(kagome::transaction_pool, TransactionPoolError, e) {
   using E = kagome::transaction_pool::TransactionPoolError;
   switch (e) {
-    case E::ALREADY_IMPORTED:
+    case E::TX_ALREADY_IMPORTED:
       return "Transaction has already been imported to the pool";
+    case E::TX_NOT_FOUND:
+      return "Transaction not found in the pool";
+    case E::POOL_IS_FULL:
+      return "Transaction pool is full";
   }
-  return "Unknown error";
```

Comment:
Still need it. Value in an enum may be invalid.

Reply:
But it's unreachable code. It possible only at memory corruption or intented type casting. Any static analizer talk about it as dead code.